python/tests/test_pyvw.py

```diff
@@ -183,3 +189,60 @@ def test_parse():
     ex = model.parse(["| a:1 b:0.5", "0:0.1:0.75 | a:0.5 b:1 c:2"])
     assert len(ex) == 2
     del model
+
+def test_numspace_id():
+    vw_ex = vw(quiet=True)
+    ex = vw_ex.example('1 |a two features |b more features here')
+    nm1 = pyvw.namespace_id(ex, 2)
+    assert nm1.id == 2
+    assert nm1.ord_ns == 128
+    assert nm1.ns == '\x80'
+    nm2 = pyvw.namespace_id(ex, '')
+    assert nm2.id is None
+    assert nm2.ord_ns == 32
+    assert nm2.ns == ' '
+
+def test_example_namespace():
+    vw_ex = vw(quiet=True)
+    ex = vw_ex.example('1 |a two features |b more features here')
+    nmid = pyvw.namespace_id(ex, 1)
+    exm = pyvw.example_namespace(ex, nmid)
+    assert isinstance(exm.ex, pyvw.example)
+    assert isinstance(exm.ns, pyvw.namespace_id)
+    assert exm.ns_hash is None
+    assert exm.num_features_in() == 3
+    assert exm[2] == (11617, 1.0)
+    assert exm.iter_features()
+    assert exm.pop_feature()
+    exm.push_features(nmid, ['c', 'd'])
+    assert exm.num_features_in() == 4
+
+def test_simple_label():
+    vw_ex = vw(quiet=True)
+    ex = vw_ex.example('1 |a two features |b more features here')
+    sl = pyvw.simple_label(2.0, weight=0.5)
```

Comment:
This is testing both a standalone simple label and one from an example- can you please split into two tests? Can you also do it for the other tests where this is the same?